lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1459,19 +1461,22 @@ def jsonrpc_claim_show(self, name=None, txid=None, nout=None, claim_id=None):
     @AuthJSONRPCServer.auth_required
     @defer.inlineCallbacks
     @AuthJSONRPCServer.flags(force='-f')
-    def jsonrpc_resolve(self, uri, force=False):
+    def jsonrpc_resolve(self, force=False, uri=None, uris=[]):
         """
         Resolve a LBRY URI
 
         Usage:
-            resolve <uri> [-f]
+            resolve (<uri> | --uri=<uri>) [<uris>...] [-f]
```

Comment:
this is weird. shouldn't it just be `resolve [-f] uris...`? What's the point of `--uri` here? Is this just to maintain BC? If so, its probably best to fix this by handling this in the function itself (e.g. if you get one uri and its of the form `uri=whatever`, then remove the `uri=`). This way people are not encouraged to use the bad usage.